app/(tabs)/products/index.tsx

```
@@ -64,9 +76,10 @@ const Product = ({
  * Products List Screen
  */
 const Products = observer(() => {
+	const productsModel$ = rootStore.productsModel;
 	useFocusEffect(
 		useCallback(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: productsModel$.getProductsList

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)